src/main/java/nopol_examples/nopol_example_3/NopolExample.java

```diff
@@ -6,13 +6,13 @@
 	 * Return true if a is odd number
 	 */
 	public boolean isOddNumber(int a){
-		int tmp = (a-1)%2;
-		int foo = 0;
+		int tmp = (a-1)%2; 
+		int foo = 0; 
 		
 		if ( tmp != 0 ){ // Fix : tmp == 0
-			return true;
+			return true; 
```

Comment:
This line (13) has been identified with a suspiciousness value of 40.16%.

<details>
     <summary>Failing tests that cover this line</summary>

- `nopol_examples.nopol_example_3.NopolExampleTest#test4`
- `nopol_examples.nopol_example_3.NopolExampleTest#test7`
- `nopol_examples.nopol_example_3.NopolExampleTest#test8`
- `nopol_examples.nopol_example_3.NopolExampleTest#test5`
- `nopol_examples.nopol_example_3.NopolExampleTest#test9`
</details>